ui/pages/keyring-snaps/snap-card/snap-card.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import SnapCard from './snap-card';
+
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
we can remove this eslint disable if we change this to...

```tsx
export default {
  title: 'Components/UI/SnapCard',
} as ComponentMeta<typeofSnapCard>;
```

Reply:
I am getting a warning that ComponentMeta is deprecated and I should use Meta.
This seems to solve the eslint error
```
import { Meta } from "@storybook/react"; // Import Meta instead of ComponentMeta

export default {
  title: 'Components/UI/SnapCard',
} as Meta<typeof SnapCard>; // Use Meta instead of ComponentMeta
```